compat/poll/poll.c

```diff
@@ -55,6 +52,9 @@
 # include <unistd.h>
 #endif
 
+/* Specification.  */
+#include "poll.h"
```

Comment:
Did you mean to change this to a quote from angled brackets?

Reply:
Yes, I actually did. The rule is that `"poll.h"` refers to a file in the local worktree, while `<poll.h>` refers to the system header. And we definitely do not want any system header here.